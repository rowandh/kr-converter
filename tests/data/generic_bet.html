
<iframe width="0" height="0" name="hfrm"></iframe>
<script language="javascript">
	function goProc(mode,uid,infotype,money){
		form = document.form1;
		form.target="hfrm";
		if (money < 1) {
			alert("1보다 큰금액만 입력하실수 있습니다");
			return;
		}
		form.infotype.value=infotype;
		form.money.value=money;
		form.user_id.value=uid;
		form.action="userinmoney_proc.asp";
		
			if(confirm(uid + "에게 " + money + "원 하자처리 하시겠습니까?")){
				form.submit();
			}		

	}
</script>

<style>
		body {background:#f5f5f5;}
		* {padding:0; margin:0;}
		input:focus,
		select:focus {
			outline:none;
		}
		.page-layout {
			height:100%;
			
		}
		
		.page-layout .bar-search {
			display:flex;
			justify-content: space-between;
			align-items: center;
			flex-wrap: wrap;
			gap:0.875rem;
			
			padding:0 0.875rem 0.875rem;
			background:#2e2f39;
		    border-bottom: 1px solid #e6e6f2;
		    box-shadow: 0 1px 4px 0 rgba(82,63,105,.15);
		}
		
		.page-layout .bar-search .title {
			padding-top:0.875rem;
			font-family:'Malgun Gothic';
			color:#c5ff88;
			font-size:1.425rem;
			font-weight:bolder;
			letter-spacing:-2px;
		}
		
		.page-layout .bar-search .title > span {
			font-size:1rem;
			letter-spacing:0;
		}
		
		.page-layout .bar-search > div:nth-child(2) {
			flex-grow:1;
			
			display:inline-flex;
    		flex-wrap: wrap;
    		justify-content: flex-end;
			gap:0.875rem;
			
			padding-top:0.875rem;
			border-top:1px solid #60931e;
		}
		
		.page-layout .bar-search > div:nth-child(2) span {
		    line-height:2rem;
		    color: #fff;
		    font-weight:bold;
		}
		
		.page-layout .bar-search > div:nth-child(2) select,
		.page-layout .bar-search > div:nth-child(2) input {
		    height: calc(2em + 0.8rem + 2px);
		    padding:0 1rem;
		    font-size: 1rem;
		    font-weight: 400;
		    height:2rem;
		    line-height:2rem;
		    color: #ccc;
		    background-color: #141529;
		    background-clip: padding-box;
		    border: 1px solid #141529;
		    border-radius: 0.25rem;
		    transition: border-color .15s ease-in-out,box-shadow .15s ease-in-out;
		}
		
		.page-layout .bar-search > div:nth-child(2) input.btn-serarch {
			color:#fff;
			font-weight:bold;
			background:#73a730 ;
			border:1px solid #629322;
		}
		
		
		.table-area {
			border:1px solid #e6e6f2;
			border-radius:1rem;
			overflow-x:auto;
			background:#fff;
			box-shadow:3px 3px 5px #e7e7e7;
			margin:1rem;
		}
		
		.table-area th {
			padding: 0 0.5rem;
			font-size:1rem;
			height:46px;
			font-weight: 600;
    		color: #2e2f39;
			background:#fff;
			border-bottom:2px solid #e6e6f2;
		}
		
		.table-area th a {
			color:#cf0b0b;
			text-decoration:none;
		}
		
		.table-area td {
			font-size:14px;
			padding:0.425rem 0.75rem;
			color:#71748d;
			white-space:nowrap;
			background:#fff;
			border-bottom:1px solid #e6e6f2;
		}
		/*
		.table-area tr:nth-child(2n) td {
			background:#f4f4f5;
			border-top:1px solid #e6e6f2;
			border-bottom:1px solid #e6e6f2;
		}
		*/
		.table-area th,
		.table-area td {
			border-right:1px solid #e6e6f2;
		}
		
		.table-area table td a {
			display:inline-block;
			
			padding:0 1rem 0 1.425rem;
			line-height:1rem;
			color:#5e8d21;
			font-size:14px;
			font-weight:bold;
			text-decoration:none;
			border-radius:0.312rem;
			background:url('../images/icon-move.png') left 50% no-repeat;
		}
		
		.table-area table td a:hover {
			filter:hue-rotate(275deg);
		}
		
		.table-area table th.double-table,
		.table-area table td.double-table {
			padding:0;
		}
		
		
		.table-area table th.double-table th {
			padding:0 0.75rem;
			border-bottom:none;
		}
		
		.table-area table td.double-table td {
			line-height:1.5rem;
			border:none;
			background:none;
		}
		
		.table-area table td.double-table tr td+td {
			border-left:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr+tr td {
			border-top:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr:nth-child(2n) td {
			background:#f9f9f9;
		}
		
		.btn-history-back {
			padding:0 1rem;
			font-size:1rem;
			height:2.625rem;
			line-height:2.625rem;
			color:#fff;
			font-weight:bold; 
			background:#37a730; 
			border-radius:0.25rem; 
			border:1px solid #629322; 
			box-shadow:1px 1px 0 #629322;
			cursor:pointer;
		}
		
		.btn-history-back:hover {
			filter:brightness(1.1);
			border:1px solid #37a730; 
			box-shadow:1px 1px 0 #37a730;
		}
		
	</style>
	
	<div class="page-layout">
		<div class="bar-search">
			<div class="title">게임기록보기</div>
			<div>
                <div>
					<input type="button" value="닫기" onclick="self.close()" class="btn-serarch" />
                </div>	
			</div>
		</div>
		
		<div class="table-area">
			<table cellspacing="0" width="100%"  border="0" >
				<tr>
                          <th width="" height="34"><p align="center">라운드ID</p></th>
                          <th width="" height="34"><p align="center">시각</p></th>
                          <th width="" height="34"><p align="center">게임 종류</p></th>
                          <th width="" height="34"><p align="center">승자(족보)</p></th>
					<th width="" height="34"><p align="center">이긴금액</p></th>
					<th class="double-table">
						상세정보
					</th>
				</tr>




                            <tr bgcolor="white">
								<td width="56" height="26"><p align="center">&nbsp;15-2-86588992</p></td>
                                <td height="26" ><p align="center">&nbsp;2025-01-01 오후 5:17:24</p></td>
                                <td ><p align="center">&nbsp;홀덤</p></td>
                                <td ><p align="center">&nbsp;<b>gjsakdjba (J 플러시)</b></p></td>
<!--								<td ><p align="center">&nbsp;92806</p></td>   -->
								<td ><p align="center">&nbsp;92,806</p></td>
								<td class="double-table">
									
									<table cellspacing="0" width="100%"  border="0">										
										<colgroup>
											<col width="200" />
											<col width="*" />
											<col width="160" />
											<col width="200" />
										</colgroup>
										<tr>
											<th height="34" style="word-break:break-all"><p align="center">참가자</p></th>
											<th height="34" style="word-break:break-all"><p align="center">족보</p></th>
											<th height="34" style="word-break:break-all"><p align="center">변동 금액</p></th>
											<th height="34" style="word-break:break-all"><p align="center">남은 잔액</p></th>
										</tr>
										
										


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>ddefyyfvh</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣K <font color='red'>♥</font>7 [K 탑]  (다이) 
<br>* 시작 : [StageNo:86588992] [Credit:185,474원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[ddefyyfvh]
<br>* 앤티: -1,000원(184,474원)
<br>* 홀 카드딜: ♣K(51) <font color='red'>♥</font>7(32) [K 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:K 탑(♣K <font color='red'>♥</font>7)]
<br>* 베팅: 다이 [0](184,474원) - 베팅순서: [0][4]
[324ms]
<br>* 커뮤니티 카드 딜: H(♣K<font color='red'>♥</font>7) C (♣J<font color='red'>♥</font>2♣7)
<br>* 커뮤니티 카드 딜: H(♣K<font color='red'>♥</font>7) C (♣J<font color='red'>♥</font>2♣7) (♣2)
<br>* 커뮤니티 카드 딜: H(♣K<font color='red'>♥</font>7) C (♣J<font color='red'>♥</font>2♣7) (♣2) (♣9)
<br>* 종료: WinMoney[0원] Credit[184,474원]
<br>* 결과: 패배 [족보:K 탑] [카드:♣K <font color='red'>♥</font>7] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">184,474</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>nlkjjvdjd</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>Q <font color='red'>◆</font>J ♣J <font color='red'>♥</font>2 ♣7 ♣2 ♣9 [J 2 투 페어]  
<br>* 시작 : [StageNo:86588992] [Credit:322,562원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[nlkjjvdjd]
<br>* 앤티: -1,000원(321,562원)
<br>* 홀 카드딜: <font color='red'>◆</font>Q(24) <font color='red'>◆</font>J(23) [Q 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:Q 탑(<font color='red'>◆</font>Q <font color='red'>◆</font>J)]
<br>* 베팅: 콜 -1,000원(320,562원) - 베팅순서: [0][3]
[5866ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>Q<font color='red'>◆</font>J) C (♣J<font color='red'>♥</font>2♣7)
<br>* 턴 시작: [플랍(1)] [족보:J 원 페어(♣J <font color='red'>◆</font>J <font color='red'>◆</font>Q ♣7 <font color='red'>♥</font>2)]
<br>* 베팅: [베팅] (10,000원) Credit(310,562원) - 베팅순서: [1][5]
[3188ms]
<br>* 베팅: 콜 -10,000원(300,562원) - 베팅순서: [1][11]
[2715ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>Q<font color='red'>◆</font>J) C (♣J<font color='red'>♥</font>2♣7) (♣2)
<br>* 턴 시작: [턴(2)] [족보:J 2 투 페어(♣J <font color='red'>◆</font>J <font color='red'>♥</font>2 ♣2 <font color='red'>◆</font>Q)]
<br>* 베팅: 콜 -14,000원(286,562원) - 베팅순서: [2][2]
[3749ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>Q<font color='red'>◆</font>J) C (♣J<font color='red'>♥</font>2♣7) (♣2) (♣9)
<br>* 턴 시작: [리버(3)] [족보:J 2 투 페어(♣J <font color='red'>◆</font>J <font color='red'>♥</font>2 ♣2 <font color='red'>◆</font>Q)]
<br>* 베팅: 콜 -8,000원(278,562원) - 베팅순서: [3][2]
[4607ms]
<br>* 종료: WinMoney[0원] Credit[278,562원]
<br>* 결과: 패배 [족보:J 2 투 페어] [카드:♣J <font color='red'>◆</font>J <font color='red'>♥</font>2 ♣2 <font color='red'>◆</font>Q]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-44,000</font></p></td>
											<td style="word-break:break-all"><p align="right">278,562</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>gjsakdjba</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>J ♣8 ♣J <font color='red'>♥</font>2 ♣7 ♣2 ♣9 [J 플러시]  (Big Blind)
<br>* 시작 : [StageNo:86588992] [Credit:200,000원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:200]
<br>* NICKNAME:[gjsakdjba]
<br>* 앤티: -1,000원(199,000원)
<br>* 홀 카드딜: <font color='red'>♥</font>J(36) ♣8(46) [J 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:J 탑(<font color='red'>♥</font>J ♣8)]
<br>* 베팅: [블라인드:BIG] [금액:1,000원] [Creadit:198,000원]
<br>* 베팅: 체크 -0원(198,000원) - 베팅순서: [0][8]
[205ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>J♣8) C (♣J<font color='red'>♥</font>2♣7)
<br>* 턴 시작: [플랍(1)] [족보:J 원 페어(♣J <font color='red'>♥</font>J ♣8 ♣7 <font color='red'>♥</font>2)]
<br>* 베팅: 체크 -0원(198,000원) - 베팅순서: [1][2]
[1956ms]
<br>* 베팅: [레이즈] (20,000원) Credit(178,000원) - 베팅순서: [1][8]
[5012ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>J♣8) C (♣J<font color='red'>♥</font>2♣7) (♣2)
<br>* 턴 시작: [턴(2)] [족보:J 2 투 페어(♣J <font color='red'>♥</font>J <font color='red'>♥</font>2 ♣2 ♣8)]
<br>* 베팅: [베팅] (14,000원) Credit(164,000원) - 베팅순서: [2][1]
[5037ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>J♣8) C (♣J<font color='red'>♥</font>2♣7) (♣2) (♣9)
<br>* 턴 시작: [리버(3)] [족보:J 플러시(♣J ♣9 ♣8 ♣7 ♣2)]
<br>* 베팅: [베팅] (8,000원) Credit(156,000원) - 베팅순서: [3][1]
[3758ms]
<br>* 종료: WinMoney[92,806원] Credit[248,806원]
<br>* 결과: 승리 [족보:J 플러시] [카드:♣J ♣9 ♣8 ♣7 ♣2]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>48,806</font></p></td>
											<td style="word-break:break-all"><p align="right">248,806</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>etn54ns</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>7 <font color='red'>♥</font>5 [7 탑]  (다이) 
<br>* 시작 : [StageNo:86588992] [Credit:621,083원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:100]
<br>* NICKNAME:[etn54ns]
<br>* 앤티: -1,000원(620,083원)
<br>* 홀 카드딜: <font color='red'>◆</font>7(19) <font color='red'>♥</font>5(30) [7 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:7 탑(<font color='red'>◆</font>7 <font color='red'>♥</font>5)]
<br>* 베팅: 다이 [0](620,083원) - 베팅순서: [0][5]
[3482ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>7<font color='red'>♥</font>5) C (♣J<font color='red'>♥</font>2♣7)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>7<font color='red'>♥</font>5) C (♣J<font color='red'>♥</font>2♣7) (♣2)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>7<font color='red'>♥</font>5) C (♣J<font color='red'>♥</font>2♣7) (♣2) (♣9)
<br>* 종료: WinMoney[0원] Credit[620,083원]
<br>* 결과: 패배 [족보:7 탑] [카드:<font color='red'>◆</font>7 <font color='red'>♥</font>5] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">620,083</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>nkjlgrsd</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>9 ♠4 ♣J <font color='red'>♥</font>2 ♣7 [J 탑]  (다이) (Small Blind)
<br>* 시작 : [StageNo:86588992] [Credit:113,152원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:100]
<br>* NICKNAME:[nkjlgrsd]
<br>* 앤티: -1,000원(112,152원)
<br>* 홀 카드딜: <font color='red'>◆</font>9(21) ♠4(3) [9 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:9 탑(<font color='red'>◆</font>9 ♠4)]
<br>* 베팅: [블라인드:SMALL] [금액:1,000원] [Creadit:111,152원]
<br>* 베팅: 체크 -0원(111,152원) - 베팅순서: [0][7]
[177ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>9♠4) C (♣J<font color='red'>♥</font>2♣7)
<br>* 턴 시작: [플랍(1)] [족보:J 탑(♣J <font color='red'>◆</font>9 ♣7 ♠4 <font color='red'>♥</font>2)]
<br>* 베팅: 체크 -0원(111,152원) - 베팅순서: [1][1]
[5318ms]
<br>* 베팅: 다이 [0](111,152원) - 베팅순서: [1][7]
[417ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>9♠4) C (♣J<font color='red'>♥</font>2♣7) (♣2)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>9♠4) C (♣J<font color='red'>♥</font>2♣7) (♣2) (♣9)
<br>* 종료: WinMoney[0원] Credit[111,152원]
<br>* 결과: 패배 [족보:J 탑] [카드:♣J <font color='red'>◆</font>9 ♣7 ♠4 <font color='red'>♥</font>2] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">111,152</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>kamvgbf</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠K ♠10 ♣J <font color='red'>♥</font>2 ♣7 [K 탑]  (다이) 
<br>* 시작 : [StageNo:86588992] [Credit:343,678원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:200]
<br>* NICKNAME:[kamvgbf]
<br>* 앤티: -1,000원(342,678원)
<br>* 홀 카드딜: ♠K(12) ♠10(9) [K 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:K 탑(♠K ♠10)]
<br>* 베팅: 콜 -1,000원(341,678원) - 베팅순서: [0][1]
[1904ms]
<br>* 커뮤니티 카드 딜: H(♠K♠10) C (♣J<font color='red'>♥</font>2♣7)
<br>* 턴 시작: [플랍(1)] [족보:K 탑(♠K ♣J ♠10 ♣7 <font color='red'>♥</font>2)]
<br>* 베팅: 체크 -0원(341,678원) - 베팅순서: [1][3]
[2288ms]
<br>* 베팅: 다이 [0](341,678원) - 베팅순서: [1][9]
[2428ms]
<br>* 커뮤니티 카드 딜: H(♠K♠10) C (♣J<font color='red'>♥</font>2♣7) (♣2)
<br>* 커뮤니티 카드 딜: H(♠K♠10) C (♣J<font color='red'>♥</font>2♣7) (♣2) (♣9)
<br>* 종료: WinMoney[0원] Credit[341,678원]
<br>* 결과: 패배 [족보:K 탑] [카드:♠K ♣J ♠10 ♣7 <font color='red'>♥</font>2] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">341,678</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>vgt6yhj7l</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>10 <font color='red'>◆</font>4 ♣J <font color='red'>♥</font>2 ♣7 [J 탑]  (다이) 
<br>* 시작 : [StageNo:86588992] [Credit:404,152원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[vgt6yhj7l]
<br>* 앤티: -1,000원(403,152원)
<br>* 홀 카드딜: <font color='red'>◆</font>10(22) <font color='red'>◆</font>4(16) [10 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:10 탑(<font color='red'>◆</font>10 <font color='red'>◆</font>4)]
<br>* 베팅: 콜 -1,000원(402,152원) - 베팅순서: [0][2]
[833ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>10<font color='red'>◆</font>4) C (♣J<font color='red'>♥</font>2♣7)
<br>* 턴 시작: [플랍(1)] [족보:J 탑(♣J <font color='red'>◆</font>10 ♣7 <font color='red'>◆</font>4 <font color='red'>♥</font>2)]
<br>* 베팅: 체크 -0원(402,152원) - 베팅순서: [1][4]
[1921ms]
<br>* 베팅: 다이 [0](402,152원) - 베팅순서: [1][10]
[2298ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>10<font color='red'>◆</font>4) C (♣J<font color='red'>♥</font>2♣7) (♣2)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>10<font color='red'>◆</font>4) C (♣J<font color='red'>♥</font>2♣7) (♣2) (♣9)
<br>* 종료: WinMoney[0원] Credit[402,152원]
<br>* 결과: 패배 [족보:J 탑] [카드:♣J <font color='red'>◆</font>10 ♣7 <font color='red'>◆</font>4 <font color='red'>♥</font>2] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">402,152</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>dh34a8ply1</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>Q ♣3 ♣J <font color='red'>♥</font>2 ♣7 [Q 탑]  (다이) 
<br>* 시작 : [StageNo:86588992] [Credit:200,000원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:200]
<br>* NICKNAME:[34vdsgfds]
<br>* 앤티: -1,000원(199,000원)
<br>* 홀 카드딜: <font color='red'>♥</font>Q(37) ♣3(41) [Q 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:Q 탑(<font color='red'>♥</font>Q ♣3)]
<br>* 베팅: 콜 -1,000원(198,000원) - 베팅순서: [0][6]
[4215ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>Q♣3) C (♣J<font color='red'>♥</font>2♣7)
<br>* 턴 시작: [플랍(1)] [족보:Q 탑(<font color='red'>♥</font>Q ♣J ♣7 ♣3 <font color='red'>♥</font>2)]
<br>* 베팅: 다이 [0](198,000원) - 베팅순서: [1][6]
[2203ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>Q♣3) C (♣J<font color='red'>♥</font>2♣7) (♣2)
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>Q♣3) C (♣J<font color='red'>♥</font>2♣7) (♣2) (♣9)
<br>* 종료: WinMoney[0원] Credit[198,000원]
<br>* 결과: 패배 [족보:Q 탑] [카드:<font color='red'>♥</font>Q ♣J ♣7 ♣3 <font color='red'>♥</font>2] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">198,000</p></td>
											
										</tr>

									 </table>
								</td>
                            </tr>






						
                        </table>
                        
                        <table cellpadding="0" cellspacing="0" border="0" width="100%">
                            <tr>
                                <td width="100%" align="center" style="padding:1rem 0; background:#fdfdfd;">
                                	<input type="button" value="닫기" onclick="self.close()" class="btn-history-back">
                                </td>
                            </tr>
                        </table>
                        
		</div>
                        
	</div>




<form name="form1" method="Post">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="logno" value="15-2-86588992">
<input type="hidden" name="regdate_f" value="">
<input type="hidden" name="user_id">
<input type="hidden" name="userid">
<input type="hidden" name="rebateid">
<input type="hidden" name="topid">
<input type="hidden" name="infotype">
<input type="hidden" name="money">
</form>

<form name="form3" method="post">
<input type="hidden" name="idx">
<input type="hidden" name="p_logno">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="page" value="">

<input type="hidden" name="s_regdate_y1" value="">
<input type="hidden" name="s_regdate_m1" value="">
<input type="hidden" name="s_regdate_d1" value="">
<input type="hidden" name="s_regdate_h1" value="">
<input type="hidden" name="game_id" value="">
<input type="hidden" name="s_regdate" value="">
</form>


