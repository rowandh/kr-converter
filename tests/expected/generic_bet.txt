PokerStars Hand #15286588992:  Hold'em No Limit ($1000/$1000) - 2025/01/01 17:17:24 KST
Table 'Table 1' 9-max Seat #6 is the button
Seat 1: kamvgbf ($343678 in chips)
Seat 2: vgt6yhj7l ($404152 in chips)
Seat 3: nlkjjvdjd ($322562 in chips)
Seat 4: ddefyyfvh ($185474 in chips)
Seat 5: etn54ns ($621083 in chips)
Seat 6: dh34a8ply1 ($200000 in chips)
Seat 7: nkjlgrsd ($113152 in chips)
Seat 8: gjsakdjba ($200000 in chips)
kamvgbf: posts the ante $1000
vgt6yhj7l: posts the ante $1000
nlkjjvdjd: posts the ante $1000
ddefyyfvh: posts the ante $1000
etn54ns: posts the ante $1000
dh34a8ply1: posts the ante $1000
nkjlgrsd: posts the ante $1000
gjsakdjba: posts the ante $1000
nkjlgrsd: posts small blind $1000
gjsakdjba: posts big blind $1000
*** HOLE CARDS ***
kamvgbf: calls $1000
vgt6yhj7l: calls $1000
nlkjjvdjd: calls $1000
ddefyyfvh: folds
etn54ns: folds
dh34a8ply1: calls $1000
nkjlgrsd: checks
gjsakdjba: checks
*** FLOP *** [Jc 2h 7c]
nkjlgrsd: checks
gjsakdjba: checks
kamvgbf: checks
vgt6yhj7l: checks
nlkjjvdjd: bets $10000
dh34a8ply1: folds
nkjlgrsd: folds
gjsakdjba: raises $10000 to $20000
kamvgbf: folds
vgt6yhj7l: folds
nlkjjvdjd: calls $10000
*** TURN *** [Jc 2h 7c] [2c]
gjsakdjba: bets $14000
nlkjjvdjd: calls $14000
*** RIVER *** [Jc 2h 7c] [2c] [9c]
gjsakdjba: bets $8000
nlkjjvdjd: calls $8000
gjsakdjba collected $92806 from pot
*** SUMMARY ***
Total pot $98000 | Rake $5194
Board [Jc 2h 7c 2c 9c]
Seat 1: kamvgbf mucked [Ks Ts]
Seat 2: vgt6yhj7l mucked [Td 4d]
Seat 3: nlkjjvdjd showed [Qd Jd] and lost
Seat 4: ddefyyfvh mucked [Kc 7h]
Seat 5: etn54ns mucked [7d 5h]
Seat 6: dh34a8ply1 mucked [Qh 3c]
Seat 7: nkjlgrsd mucked [9d 4s]
Seat 8: gjsakdjba showed [Jh 8c] and won ($92806)